XamarinCommunityToolkitSample/Pages/Effects/SafeAreaEffectPage.xaml

```diff
@@ -9,18 +9,20 @@
     BackgroundColor="{StaticResource NormalButtonBackgroundColor}">
     <ContentPage.Content>
         <StackLayout
+            x:Name="stack"
             Padding="10,10"
-            effects:SafeAreaEffect.SafeArea="{Binding Path=IsToggled}"
             BackgroundColor="{StaticResource AppBackgroundColor}"
-            BindingContext="{x:Reference Name=ActivationToggle}"
```

Comment:
Why was this changed?)

Reply:
because this was throwing an exception. So I changed it to use the codebehind

![image](https://user-images.githubusercontent.com/20712372/96347518-89443280-1078-11eb-85a4-92d0a0913d84.png)

Reply:
That's strange. It worked for me before and I guess we want it to work through binding as well, right? But maybe have a look at that separately

Reply:
@jfversluis I think that we can create a converter or something like that